DuckDuckGoTests/ContextualDaxDialogsFactoryTests.swift

```diff
@@ -353,7 +353,7 @@ extension ContextualDaxDialogsFactoryTests {
         XCTAssertNotNil(host.view)
 
         // THEN
-        waitForExpectations(timeout: 2.0)
+        waitForExpectations(timeout: 10.0)
```

Comment:
It's not ideal to have to wait this much time, but it's cut short if the test succeeds earlier so it should be more or less fine.

Locally this test succeeds in around 1.2 seconds.